test/issue-strategies/story-test.js

```diff
@@ -16,6 +19,21 @@ describe('Story/Sub-task Strategy Apply Tests', () => {
     it('Story is yellow and parent is an initiative which is also yellow', () =>
       storyStrat.apply(dummyJira.issues.Story2, dummyJira).should.eventually.equal(true)
     );
+
+    it('Story is a child of a Sub-task from the dispatcher and all parents are yellow', ()=>
+      storyStrat.apply(dummyJira.issues.LinkedStory1, dummyJira).should.eventually.equal(true)
+    );
+
+    it('Sub-task has a parent dispatcher, is linked to a story, and all parents are yellow', ()=>
+      storyStrat.apply(dummyJira.issues.LinkedSubtask1, dummyJira).should.eventually.equal(true));
+    it('Sub-task has a parent Maintainance task, and all parents are yellow', ()=>
+      storyStrat.apply(dummyJira.issues.LinkedSubtask2, dummyJira).should.eventually.equal(true));
+    it('Sub-task has a parent Bug, and all parents are yellow', ()=>
+      storyStrat.apply(dummyJira.issues.LinkedSubtask3, dummyJira).should.eventually.equal(true));
+    it('Sub-task has a parent Bug, with an epic link', ()=>
+      storyStrat.apply(dummyJira.issues.BugSubtask2, dummyJira).should.eventually.equal(true));
+    it('Sub-task has a parent Bug, with an epic link', ()=>
+      storyStrat.apply(dummyJira.issues.BugSubtask2, dummyJira).should.eventually.equal(true));
```

Comment:
The lack of spaces between tests really threw me off, but great use of `chai-as-promised`. We could probably make this even more succinct by using `should.eventually.be.true` instead of `equal`. Let's format this as follows:
```js
it('test description', () =>
  //assertion
);

it('test description', =>
  //assertion
);
```

The space separation makes this much easier to read and differentiate one test from another.